elections/models.py

```diff
@@ -227,7 +226,7 @@ def group_and_order_elections(cls, include_posts=False,
                 'election': election
             }
             if include_posts:
-                d['posts'] = list(election.posts.all())
+                d['posts'] = list(election.postextraelection_set.all())
```

Comment:
I'm assuming then it would make sense to change the include_posts parameter name too.